src/js/view/ticket-list.js

```diff
@@ -0,0 +1,50 @@
+export class TicketList {
+  #element;
+  #ticketHTMLString = `<li class="mx-1 text-4xl lotto-wrapper">
+  <span class="lotto-icon">🎟️ </span>
+  <span class="lotto-detail" style="display: none;"></span>
+</li>`;
```

Comment:
이건 상수로 만들어서 관리해주면 어떨까요?
```suggestion
  static #TICKET_TEMPLATE = `
    <li class="mx-1 text-4xl lotto-wrapper">
      <span class="lotto-icon">🎟️ </span>
      <span class="lotto-detail" style="display: none;"></span>
    </li>
  `;
```